com.unity.shadergraph/Editor/GraphDeltaRegistry/Generation/Interpreter.cs

```diff
@@ -290,13 +289,13 @@ private static bool FunctionsAreEqual(ShaderFunction a, ShaderFunction b)
 
         private static string ConvertToFloat3(ShaderType type, string name)
         {
-            switch(type.VectorDimension)
+            return type.VectorDimension switch
             {
-                case 4: return $"{name}.xyz";
-                case 2: return $"float3({name}.x, {name}.y, 0)";
-                case 1: return $"float3({name}, {name}, {name})";
-                default: return name;
-            }
+                4 => $"{name}.xyz",
+                2 => $"float3({name}.x, {name}.y, 0)",
+                1 => $"float3({name}, {name}, {name})",
+                _ => name,
```

Comment:
This syntax exists?

Reply:
Yep! It's new to me too. However, this is the MS suggested syntax for switch statements that have a break for every case.

Reply:
Honestly, it surprises me that anonymous function creation is this cheap.